### ISSUE TITLE
Unparsable value for LS_COLORS environment variable

### ISSUE THREAD

**Issue by FredDeschenes** [fish-shell/fish-shell#3563]
- [x] Have you checked if problem occurs with [fish 2.4.0](/fish-shell/fish-shell/releases/tag/2.4.0)?
- [x] Tried fish without third-party customizations *(check `sh -c 'env HOME=$(mktemp -d) fish'`)*?

**fish version installed** *(`fish --version`)*: fish, version 2.4.0

**OS/terminal used**: Cygwin on Windows 8.1 and iTerm2 on OSX Sierra
Both started doing this after I updated to fish 2.4.0

## Reproduction steps
1. Open terminal
2. type "ls"

## Results
```console
~ $ ls
ls: unparsable value for LS_COLORS environment variable
# contents printed here, but without color
~ $ echo $LS_COLORS
rs=0:di=01;34:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:mi=00:su=37;41:sg=30;43:ca=30;41:tw=30;42:ow=34;42:st=37;44:ex=01;32:*.tar=01;31:*.tgz=01;31:*.arc=01;31:*.arj=01;31:*.taz=01;31:*.lha=01;31:*.lz4=01;31:*.lzh=01;31:*.lzma=01;31:*.tlz=01;31:*.txz=01;31:*.tzo=01;31:*.t7z=01;31:*.zip=01;31:*.z=01;31:*.Z=01;31:*.dz=01;31:*.gz=01;31:*.lrz=01;31:*.lz=01;31:*.lzo=01;31:*.xz=01;31:*.bz2=01;31:*.bz=01;31:*.tbz=01;31:*.tbz2=01;31:*.tz=01;31:*.deb=01;31:*.rpm=01;31:*.jar=01;31:*.war=01;31:*.ear=01;31:*.sar=01;31:*.rar=01;31:*.alz=01;31:*.ace=01;31:*.zoo=01;31:*.cpio=01;31:*.7z=01;31:*.rz=01;31:*.cab=01;31:*.jpg=01;35:*.jpeg=01;35:*.gif=01;35:*.bmp=01;35:*.pbm=01;35:*.pgm=01;35:*.ppm=01;35:*.tga=01;35:*.xbm=01;35:*.xpm=01;35:*.tif=01;35:*.tiff=01;35:*.png=01;35:*.svg=01;35:*.svgz=01;35:*.mng=01;35:*.pcx=01;35:*.mov=01;35:*.mpg=01;35:*.mpeg=01;35:*.m2v=01;35:*.mkv=01;35:*.webm=01;35:*.ogm=01;35:*.mp4=01;35:*.m4v=01;35:*.mp4v=01;35:*.vob=01;35:*.qt=01;35:*.nuv=01;35:*.wmv=01;35:*.asf=01;35:*.rm=01;35:*.rmvb=01;35:*.flc=01;35:*.avi=01;35:*.fli=01;35:*.flv=01;35:*.gl=01;35:*.dl=01;35:*.xcf=01;35:*.xwd=01;35:*.yuv=01;35:*.cgm=01;35:*.emf=01;35:*.ogv=01;35:*.ogx=01;35:*.aac=00;36:*.au=00;36:*.flac=00;36:*.m4a=00;36:*.mid=00;36:*.midi=00;36:*.mka=00;36:*.mp3=00;36:*.mpc=00;36:*.ogg=00;36:*.ra=00;36:*.wav=00;36:*.oga=00;36:*.opus=00;36:*.spx=00;36:*.xspf=00;36: >&/dev/null
```

This seems to have been caused by bb754c20858bcd911d96c9933e0cecc9a9acbb75. For some reason I still get the `>&/dev/null` output at the end of `dircolors -c $colorfile`. I do not have a `~/.dir_colors` or `~/.dircolors` file and have not modified the `/etc/DIR_COLORS` that's bundled with Cygwin/OSX.

**Comment by FredDeschenes** [comment]
I also just realized that [this PR](https://github.com/fish-shell/fish-shell/pull/3560) talks about this, forgot to turn off the "is:issue" filter when I searched, sorry if this is already fixed!

**Comment by krader1961** [comment]
No need to apologize for not searching PRs, @FredDeschenes. This is a good example for why I feel strongly that every non-trivial PR should have an associated issue. @c2v4, when you update your PR please also add a "Fixes #3563" line to the commit comment.
